server/chat/backend/agent/tools/cloud_tools.py

```
@@ -690,7 +753,47 @@ def wrapper(*args, **kwargs):
+                    _sctx = get_state_context()
+                    _sid = _sctx.session_id if _sctx and hasattr(_sctx, 'session_id') else None
+                except Exception as ctx_err:
+                    logging.warning(
+                        f"Output-redaction context lookup failed for {tool_name}: {ctx_err}; "
+                        "scanning with empty user/session metadata"
+                    )
+                try:
+                    if isinstance(result, (dict, list)):
+                        try:
+                            result = json.dumps(result, ensure_ascii=False, default=str)
+                        except Exception as dump_err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)